lib/suse/connect/cli.rb

```diff
@@ -149,6 +149,10 @@ def extract_options # rubocop:disable MethodLength
           @options[:write_config] = true
         end
 
+        @opts.on('--cleanup', 'remove old system credentials and all zypper services installed by SUSEConnect') do |opt|
```

Comment:
Should this be 'zypp' rather than 'zypper' so that it is clear that the services are not restricted to zypper?